contrib/kubernikus-tests/Dockerfile

```diff
@@ -5,12 +5,13 @@ WORKDIR /go/src/github.com/sapcc/kubernikus/
 RUN apk add --no-cache make git curl bash
 
 RUN curl -Lf https://storage.googleapis.com/kubernetes-helm/helm-v2.10.0-linux-amd64.tar.gz \
-		| tar --strip-components=1 -C /usr/local/bin -zxv \
-		&& helm version -c
```

Comment:
Why remove the binary invocation. The purpose of those are to check that download/upack indeed moves a valid binary in place

Reply:
@databus23 True, i added it to the other binaries as well.